### ISSUE TITLE
Incorrect output locations

### ISSUE THREAD

**Issue by RainDogSoftware** [RainDogSoftware/pingpung#1]
Description:  Output which should be going to a particular tabs winds up being displayed on its intended tab as well as other tabs.  The output is sent by ShowResult method and contains a TabID indicating which integer-numbered tab should receive the data and display the output.  It seems that results are _sometimes_ being duplicated on multiple tabs.  

Output has been adjusted to include TabID with each output display call, and confirms that the output contains the tabID of the "wrong" tab.  

In short, output is sometimes duplicated to tabs other than the intended.  My guess is that it has something to do with timing, as I occasionally see things like "Every 4th ping from Tab2 displays on Tab 2 and Tab 1".  Mostly it's sporadic.  

To reproduce:  Run app with python3/qt4 as administrator, start different pings on different tabs.  Will run fine for a while but at some point you'll see results intended for tab2 on tab1, etc.
